drivers/lora/sx1276.c

```diff
@@ -93,61 +99,98 @@ void SX1276Reset(void)
 	k_sleep(6);
 }
 
-void BoardCriticalSectionBegin(uint32_t *mask)
+void BoardCriticalSectionBegin(u32_t *mask)
 {
 	*mask = irq_lock();
 }
 
-void BoardCriticalSectionEnd(uint32_t *mask)
+void BoardCriticalSectionEnd(u32_t *mask)
 {
 	irq_unlock(*mask);
 }
 
-uint32_t RtcGetTimerElapsedTime(void)
+u32_t RtcGetTimerValue(void)
 {
-	u32_t ticks;
-	int err;
-
-	err = counter_get_value(dev_data.counter, &ticks);
-	if (err) {
-		LOG_ERR("Failed to read counter value (err %d)", err);
-		return 0;
-	}
+	return k_ms_to_ticks_ceil32(k_uptime_get_32());
```

Comment:
Why don't you use `k_cycle_get_32` directly here?

Reply:
I tried it but the calculated delta between two points doesn't seem to be right. Seems like it takes accumulated count into account.